modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -2,23 +2,37 @@
 //
 // persistent-storage-csi-vsphere.adoc
 //
-
+:_content-type: PROCEDURE
 [id="persistent-storage-csi-vsphere-install-issues_{context}"]
 = Removing a third-party vSphere CSI Operator Driver
 
-{product-title} 4.10 includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat. If you have installed a vSphere CSI driver provided by the community or another vendor, upgrades will be prevented in a future release of {product-title}.
+{product-title} {product-version} includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat.
+
+If you have installed a vSphere CSI driver provided by the community or another vendor, which is considered a third-party vSphere CSI driver, and you continue with upgrading to the next major version of {product-title}, the `oc` CLI prompts you with the following message:
+
+[source,terminal]
+----
+VSphereCSIDriverOperatorCRUpgradeable: VMwareVSphereControllerUpgradeable:
+found existing unsupported csi.vsphere.vmware.com driver
+----
+
+The previous message informs you that Red Hat does not support the third-party vSphere CSI driver during an {product-title} upgrade operation. You can choose to ignore this message and continue with the upgrade operation.
 
-If you remove the third-party vSphere CSI driver, you do not need to delete associated persistent volume (PV) objects, and no data loss should occur.
+The instructions outlined in the procedure show how to uninstall a third-party vSphere CSI Driver. Consult the vendor's or community provider's uninstall guide for more detailed instructions on removing the driver and its components.
 
-[NOTE]
+[IMPORTANT]
 ====
-These instructions may not be complete, so consult the vendor or community provider uninstall guide to ensure removal of the driver and components.
+When removing a third-party vSphere CSI driver, you do not need to delete the associated persistent volume (PV) objects. Data loss typically does not occur, but Red Hat does not take any responsibility if data loss does occur.
 ====
 
-To uninstall the third-party vSphere CSI Driver:
+After you have removed the third-party vSphere CSI Driver from the {product-title} cluster, installation of Red Hat's vSphere CSI Operator Driver automatically resumes. If you had existing vSphere CSI PV objects, their lifecycle is now managed by Red Hat's vSphere CSI Operator Driver.
+
+.Procedure
 
 . Delete the third-party vSphere CSI Driver (VMware vSphere Container Storage Plugin) Deployment and Daemonset objects.
+
 . Delete the configmap and secret objects that were installed previously with the third-party vSphere CSI Driver.
+
 . Delete the third-party vSphere CSI driver `CSIDriver` object:
 +
 [source,terminal]
```

Comment:
@dfitzmau This comment is outside of the scope of the PR (so I merged it already), but maybe for a follow-up, I did notice in the preview build that this block here has a `~` before the `$` prompt for some reason. And the block immediately following it could use a `.Example output` title.